SConstruct

```diff
@@ -40,6 +40,7 @@ includePath = [ sourceRoot,
 
 linkPath    = [ os.environ['CLHEP_LIB_DIR'],
                 os.environ['ROOTSYS'] + '/lib',
+                '#/lib',
```

Comment:
Isn't there a dangling comma here?  Or maybe the # is intended to be outside the ''?  If so, then there's still a dangling comma from the line above.